sdk/communication/ci.yml

```diff
@@ -30,8 +30,6 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: communication
-    MatrixFilters:
```

Comment:
The filters still keep the test agents from provisioning and the live test resources from being deployed, so it saves a bit of time and $.